devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -147,6 +162,14 @@ def installed() {
 def updated() {
 	sendHubCommand(pause())
 	state.reverseDirection = reverseDirection ? reverseDirection : false
+    int time = calibrationTime
```

Comment:
Please use tabs instead of spaces for indentation. (There are some spaces in the preference definition block above, as well.)

Reply:
@dkirker - fixed